javascripts/downloads.js

```diff
@@ -3,29 +3,34 @@ function onLoad() {
     var sVersionTemplate = $("#versionTemplate").html();
     var sStableVersionTemplate = $("#stableVersionTemplate").html();
     var sUnstableVersionTemplate = $("#unstableVersionTemplate").html();
+    var sNoUnstableVersionTemplate = $("#noUnstableVersion").html();
     var oAvailableVersionsElement = $("#availableVersions");
     var oStableVersionElement = $("#stableVersion");
     var oUnstableVersionElement = $("#unstableVersion");
+    var oNoUnstableVersionElement = $("#noUnstableVersion");
     //last stable release object
     var oStableVersion = null;
     //unstable development build object
     var oUnstableVersion = null;
 
     jQuery.getJSON("./OpenUI5Downloads.json", function (oResult) {
+	var bBeta = false;
```

Comment:
mixture of tabs and spaces, replace "    "  with \t or vice-versa

Reply:
Michadelic, I don't understand what you meant here.